address/src/main/java/com/bloxbean/cardano/client/address/AddressProvider.java

```diff
@@ -375,7 +375,7 @@ public static Address getRewardAddress(@NonNull Credential stakeCredential, Netw
         }
     }
 
-    private static Address getAddress(byte[] paymentKeyHash, byte[] stakeKeyHash, byte headerKind, Network networkInfo, AddressType addressType) {
+    public static Address getAddress(byte[] paymentKeyHash, byte[] stakeKeyHash, byte headerKind, Network networkInfo, AddressType addressType) {
```

Comment:
@stikkos Rather than exposing this method, could you use the following method, which accepts Credential objects as parameters? I think that this method is too low-level, as developers might pass an incorrect headerKind byte.

```
Address getBaseAddress(Credential paymentCredential, Credential delegationCredential, Network networkInfo) 
```

Similar methods exist for creating Enterprise  and Reward addresses.

Reply:
Yes, great. I can use this method indeed. Have reverted the visibility change